docs/data/sql_functions.yml

```diff
@@ -617,6 +617,12 @@ collection:
   - sql: ARRAY_DISTINCT(haystack)
     table: haystack.arrayDistinct()
     description: Returns an array with unique elements. If the array itself is null, the function will return null. Keeps ordering of elements.
+  - sql: ARRAY_REMOVE(haystack, needle)
+    table: haystack.arrayRemove(needle)
+    description: Remove all elements that equal to element from array. If the array itself is null, the function will return null.
+  - sql: ARRAY_SIZE(haystack)
+    table: haystack.arraySize()
+    description: Returns the size of an array. If the array itself is null, the function will return null.
```

Comment:
I still don't understand why `ARRAY_SIZE` is required if there is already `CARDINALITY` which does same thing.
There is a discussion in jira about that

Reply:
To align the other engines like spark,snowflake, so that users can migrate easily

Reply:
thanks for your details response, i will drop the array_size commit.